vendor/vendor.json

```diff
@@ -2,6 +2,108 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
+			"checksumSHA1": "2qnf4VbkRU+k/c7xFjL1CQ0ywpE=",
+			"path": "docker.io/go-docker",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "OljfuXdUfmwq8tRL+5nnMGBv7b4=",
+			"path": "docker.io/go-docker/api",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "BkzCLaKc6Xe+aa6/s3zf4noj0qI=",
+			"path": "docker.io/go-docker/api/types",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "qqB8ovIl+cvmP5296eN4YtZOF1M=",
+			"path": "docker.io/go-docker/api/types/blkiodev",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "Y6ZHehxkNE9jUz3JUUjiFJDMkJQ=",
+			"path": "docker.io/go-docker/api/types/container",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "SEztk2u2Tg0eMWE+kDQH6GHNbVM=",
+			"path": "docker.io/go-docker/api/types/events",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "h/cirDA2cSjGva4IedkSknceMzQ=",
+			"path": "docker.io/go-docker/api/types/filters",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "O+x4U86tDEYYDnp0aMSLm+Em76Q=",
+			"path": "docker.io/go-docker/api/types/image",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "nJgyIP7/0GIw4tk1kW8eNs7Gdpg=",
+			"path": "docker.io/go-docker/api/types/mount",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "E3/rd0Z/3X3pM27YqO9VZ+GlNcQ=",
+			"path": "docker.io/go-docker/api/types/network",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "fki/ItBkrrQ/+X7W6hH/2y6JhMg=",
+			"path": "docker.io/go-docker/api/types/registry",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
+			"revisionTime": "2017-09-28T21:45:39Z"
+		},
+		{
+			"checksumSHA1": "N4QuNyYs3ToBqV6cR6lO6rvAJkE=",
+			"path": "docker.io/go-docker/api/types/strslice",
+			"revision": "4daae26030ad00e348edddff9767924ae57a3b82",
```

Comment:
I was initially curious if we should stick to a release here. Looking at https://github.com/docker/go-docker I looks like there are only 7 commits in this repo an all 8 month old. Is this really the lib that docker recommend moving forward?

Reply:
as I said, that lib is still work in progress, there is quite a mess with docker/moby and vendoring, I opted for keep using our current client, but implement negotiation with it, see #7165